src/Empire/Empire.Api.ts

```diff
@@ -172,11 +170,20 @@ export class EmpireApi {
         Memory.flags[flag.name].flagType = flagTypeConst;
         Memory.flags[flag.name].flagName = flag.name;
 
+        // Delete all creep associated with remote room
         const ownedRooms: Room[] = MemoryApi_Empire.getOwnedRooms();
         delete Memory.rooms[remoteRoomName];
         for (const room of ownedRooms) {
             if (!room.memory.remoteRooms) room.memory.remoteRooms = {};
             delete Memory.rooms[room.name].remoteRooms![remoteRoomName];
         }
+
+        // Suicide all creeps associated with the remote room
+        for (let i in Game.creeps) {
```

Comment:
Is checking just targetRoom enough here? Is there any risk that a creep traveling through the area could be accidentally suicided by this?